engine/src/main/java/org/hibernate/search/spi/SearchIntegratorBuilder.java

```diff
@@ -499,37 +497,6 @@ private static IndexingMode defineIndexingMode(SearchConfiguration cfg) {
 		return IndexingMode.fromExternalRepresentation( indexingStrategy );
 	}
 
-	private ErrorHandler createErrorHandler(SearchConfiguration searchConfiguration) {
-		Object configuredErrorHandler = searchConfiguration.getProperties().get( Environment.ERROR_HANDLER );
```

Comment:
> although a Service is 1 per integrator

I was with the impression hsearch services were based on ```java.util.ServiceLoader``` and thus 1 instance only for across all SearchIntegrator...

> Do you need to handle some error internally in Infinispan, while still forward to the user other errors?

Yes. On Infinispan, during topology changes, there is a small time window where a lock obtention fails due to transient ownership changes, and my custom error handler catches that and redirect the work to the rightful owner.